dev/io.openliberty.microprofile.internal_fat/fat/src/io/openliberty/microprofile/internal/test/suite/MPCompatibleTest.java

```diff
@@ -0,0 +1,346 @@
+    public void testMP40andCtxPropagtion10() throws Exception {
+        try {
+            server.setServerConfigurationFromFilePath("MP40andMPCtx10.xml");
+            server.startServerAndValidate(true, //preClean
+                                          false, //cleanStart
+                                          false, //validateApps
+                                          false, //expectStartFailure
+                                          false); //validateTimedExit
+        } finally {
+            server.stopServer("CWWKF0033E: The singleton features io.openliberty.mpCompatible-0.0 and io.openliberty.mpCompatible-4.0 cannot be loaded at the same time.",
+                              "CWWKF0046W: The configuration includes an incompatible combination of features");
+        }
+    }
+
+    /**
+     * microProfile-4.0 plus mpContextPropagation-1.1
+     * Should always pass
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.LITE)
+    public void testMP40andCtxPropagtion11() throws Exception {
+        try {
+            server.setServerConfigurationFromFilePath("MP40andMPCtx11.xml");
+            server.startServer();
+            runGetMethod(200, "/helloworld/helloworld", MESSAGE);
+        } finally {
+            server.stopServer("CWMOT0010W"); //CWMOT0010W: OpenTracing cannot track JAX-RS requests because an OpentracingTracerFactory class was not provided or client libraries for tracing backend are not in the class path.
+        }
+    }
+
+    /**
+     * microProfile-3.3 plus mpGraphQL-1.0 and mpReactiveMessaging-1.0
+     * Should pass.
+     *
+     * mpGraphQL-1.0 currently produces some warnings that need to be investigated
+     * https://github.com/OpenLiberty/open-liberty/issues/15496
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    @Ignore //TODO: re-enable when issue 15496 has been resolved
+    public void testMP33plusStandalone() throws Exception {
+        try {
+            server.setServerConfigurationFromFilePath("MP33plusStandalone.xml");
+            server.startServer();
+            runGetMethod(200, "/helloworld/helloworld", MESSAGE);
+        } finally {
+            server.stopServer("CWMOT0010W"); //CWMOT0010W: OpenTracing cannot track JAX-RS requests because an OpentracingTracerFactory class was not provided or client libraries for tracing backend are not in the class path.
+        }
+    }
+
+    /**
+     * microProfile-4.0 plus mpGraphQL-1.0
+     * mpReactiveMessaging-1.0 should be included once the following feature is completed
+     * https://github.com/OpenLiberty/open-liberty/issues/15440
+     *
+     * mpGraphQL-1.0 currently produces some warnings that need to be investigated
+     * https://github.com/OpenLiberty/open-liberty/issues/15496
+     *
+     * Should pass.
+     *
+     * @throws Exception
+     */
+    @Test
+    @Mode(TestMode.FULL)
+    @Ignore //TODO: re-enable when issue 15496 has been resolved
```

Comment:
Should you enable this test but add the server error ignore and then add a comment to that issue? When the issue is fixed, the warning can be removed.